jp2_online/settings/base.py

```diff
@@ -177,3 +177,10 @@
 STATIC_URL = '/static/'
 
 LOGIN_URL = 'tosp_auth:login'
+
+EMAIL_HOST = 'smtp.sendgrid.net'
+EMAIL_HOST_USER = 'isjp2.thelma'
+EMAIL_HOST_PASSWORD = 'juanPablo2'
```

Comment:
This is not obligatory for this code review, but we should consider moving this to enviornment variables. Am sure someone is going to open an issue in the future.